File: VB/VB/MyLayoutControl/MySkinGroupObjectPainter.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraLayout.Registrator
Imports DevExpress.XtraLayout
Imports DevExpress.LookAndFeel
Imports DevExpress.XtraLayout.Painting
Imports DevExpress.XtraLayout.ViewInfo
Imports DevExpress.Utils.Drawing
Imports DevExpress.Skins

Namespace WindowsApplication1
	Public Class MySkinGroupObjectPainter
		Inherits SkinGroupObjectPainter
		Public Sub New(ByVal owner As IPanelControlOwner, ByVal provider As ISkinProvider)
			MyBase.New(owner, provider)

		End Sub

		Public Overrides Overloads Sub DrawObject(ByVal e As ObjectInfoArgs)
			MyBase.DrawObject(e)
		End Sub

	End Class
End Namespace

File: CS/C#/MyLayoutControl/MySkinGroupObjectPainter.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraLayout.Registrator;
using DevExpress.XtraLayout;
using DevExpress.LookAndFeel;
using DevExpress.XtraLayout.Painting;
using DevExpress.XtraLayout.ViewInfo;
using DevExpress.Utils.Drawing;
using DevExpress.Skins;

namespace WindowsApplication1
{
    public class MySkinGroupObjectPainter : SkinGroupObjectPainter
    {
        public MySkinGroupObjectPainter(IPanelControlOwner owner, ISkinProvider provider)
            : base(owner, provider)
        {

        }

        public override void DrawObject(ObjectInfoArgs e)
        {
            base.DrawObject(e);
        }

    }
}
